com.unity.shadergraph/Editor/Data/Nodes/AbstractMaterialNode.cs

```diff
@@ -7,6 +7,7 @@
 using UnityEditor.ShaderGraph.Internal;
 using UnityEditor.ShaderGraph.Drawing;
 using UnityEditor.ShaderGraph.Serialization;
+using UnityEngine.Assertions;
```

Comment:
Is this used?

Reply:
There's still one unresolved bug, I'll clean these up in that PR.